syntax = "proto3";

package workflows;

import "google/protobuf/empty.proto";

service WorkflowLogService {
  rpc CreateWorkflowLog (CreateWorkflowLogRequest) returns (google.protobuf.Empty); // Return Empty
}

message CreateWorkflowLogRequest {
  string exec_id = 1;
  string type = 2;
  int64 event_time_nano = 3;
  string state = 5; // JSON data as a string
}